cfn/JavaScriptESDK.yml

```diff
@@ -0,0 +1,101 @@
+Outputs:
+  StackArn:
+    Description: >-
+      Do not remove this output! Pipelines needs this to do its association. (And
+      LPT. Removing it will break things)
+    Value: !Ref 'AWS::StackId'
```

Comment:
Suggestion: If we maintain the CFN here, we cannot use BONES pipelines to maintain the Stack.
```suggestion
```